### PR TITLE
created work about section

### PR DESCRIPTION
**Pull request by hasnatali1947**
Added work section in Html,
Added about section in Html,
Added their styling in css, 
Added icons for these sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -186,32 +186,32 @@ <h3 class="Connect-heading">LET'S CONNECT</h3>
...
-        <ul class="Multi-2-HCJ">
-          <div class="Multi-2-html">html</div>
-          <div class="Multi-2-css">css</div>
-          <div class="Multi-2-javaScript">javaScript</div>
+        <ul class="Multi-HCJ">
+          <div class="Multi-html">html</div>
+          <div class="Multi-css">css</div>
+          <div class="Multi-javaScript">javaScript</div>
         </ul>
         <button class="Button-SeeProject-4">See Project</button>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
image

File: index.html [hunk 2]
```diff
@@ -152,26 +152,26 @@ <h3 class="Connect-heading">LET'S CONNECT</h3>
...
 
-        <ul class="tonic-2-HCJ">
-          <div class="tonic-2-html">html</div>
-          <div class="tonic-2-css">css</div>
-          <div class="tonic-2-javaScript">javaScript</div>
+        <ul class="tonic-1-HCJ">
+          <div class="tonic-1-html">html</div>
+          <div class="tonic-1-css">css</div>
+          <div class="tonic-1-javaScript">javaScript</div>
         </ul>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Nice to use the `ul` tag here. But don't you use the list tag (`li`) to list the `HTML`, `CSS`, and `JavaScript`?.

File: style.css [hunk 3]
```diff
@@ -137,26 +138,35 @@ header {
...
   }
 
   .Tonic-1 a {
     text-decoration: none;
     color: #172b4d;
   }
 
+  .tonic {
+    margin-left: 1.5vh;
+  }
```
Comment (Owusu-Desmond) [inline style.css]: - [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design.
  - you have horizontal overflow so please make sure you remove the horizontal overflow.
  - make sure that you adjust the margin of each project card to look exactly like the Figma design.

| Your Design |Figma Design |
| :---:   | :---: | 
| Screenshot from 2023-02-18 16-18-50| image |

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @hasnatali1947 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,124 +1,116 @@
...
+          I'm Hasnat <br />
+          Glad to see <br />
+          you!
+        </h1>
+        <p class="First-heading-description">
+          I'm a software developer! I can help you build a prduct,feature or
+          website Look through some of my work and experience! if you like
+          what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
```
Comment (algerina) [inline index.html:39]: - [ ] You did well implementing most of the requirements however, the header is not displayed and the cards are not displayed properly as you can see in the caption. Kindly fix the issue.
- [ ] The button on hover still not matching the Figma design 

review

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @hasnatali1947 ,

Good job so far!  But still some minor issues to complete.

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- No linters errors:heavy_check_mark:



## STATUS: REQUIRED CHANGES 




 ##required changes:

- Please check the comments under the review

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html (hunk 4 above)
Comment (hasnatali1947) [reply index.html:39]: hi.. can you tell me Please from where I found hower color which matching the figma design

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @hasnatali1947 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**